### ISSUE TITLE
Not working for Old School Essentials

### ISSUE THREAD

**Issue by Naroek56** [death-save/combat-carousel#83]
OS: Windows 10
Foundry: 0.8.8
Old-School Essentials: 1.2.2

When starting combat the carousel field is empty and invisible until hovered over.
![image](https://user-images.githubusercontent.com/49316726/125927242-ad13aed2-38aa-4cdd-8d18-76add1699d97.png)

The error message reads:
```
combat-carousel.mjs:248 Uncaught (in promise) TypeError: Cannot set property 'title' of null
    at CombatCarousel.prepareTurnData (combat-carousel.mjs:248)
    at hooks.mjs:143
    at Function._call (foundry.js:194)
    at Function.callAll (foundry.js:153)
    at ClientDatabaseBackend.callback (foundry.js:8681)
    at foundry.js:8651
    at Array.map (<anonymous>)
    at ClientDatabaseBackend._handleCreateEmbeddedDocuments (foundry.js:8651)
    at ClientDatabaseBackend._createEmbeddedDocuments (foundry.js:8544)
    at async Function.createDocuments (document.mjs:332)
```
Fix/help much appreciated!

**Comment by eclarke12** [comment]
Same cause as #98

**Comment by eclarke12** [comment]
Fixed by #100
